docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -734,54 +734,130 @@ Baseline coverage of treatment for elevated SBP and elevated LDL-c is substantia
   * - 6.0
     - Adding in the outreach intervention 
     -  
+
+Model 3 V&V for the relative risk with angina showed a lot of variability: 
+    .. image:: Model3_VV_Angina.png
+
   
 .. _uscvd4.7:
 
 4.7 Desired outputs
 -------------------
- .. todo::
-    Validate the below with project partners: 
-
-    - Are DALYs cause specific or is total DALYs okay? 
-    - Is change in risk factor by state also important? (i.e., a XX drop in SBP)
-    - What is missing from the below? 
 
 Outputs:
 
-#. DALYs (YLLs and YLDs) due to cause specific mortality 
-#. Average values for risk factors by state and year (SBP, LDL-C, FPG, and BMI)
-#. Numbers of interventions administered per a) 100,000 population, and b) 100,000 person years 
+#. Total population 
+#. Person-time 
+#. YLLs and YLDs
+#. Deaths 
+#. Transitions for each cause 
+#. Exposure by person time for all risk factors 
```

Comment:
This would be for categorical risk factors only, right? Otherwise we're tracking mean exposure values? I think we should try to be really specific here!

Reply:
Or oops, maybe I just misinterpreted "exposure by person time" and you do mean sum of all exposure values multiplied by person-time spent with that exposure value summed across all simulants to calculate the mean exposure value. Either way, I think a bit more detail to clarify would be helpful :)

Reply:
Makes sense! Added